scenarioo-server/src/main/java/org/scenarioo/business/diffViewer/comparator/ScreenshotComparator.java

```diff
@@ -18,167 +18,149 @@
-			final ComparisonConfiguration comparisonConfiguration) {
+								final ComparisonConfiguration comparisonConfiguration) {
 		super(baseBranchName, baseBuildName, comparisonConfiguration);
-		gmConsole = new CompareCmd(true);
-		gmConsoleErrorConsumer = new ArrayListErrorConsumer();
-		gmConsoleOutputConsumer = new ArrayListOutputConsumer();
-		gmConsole.setErrorConsumer(gmConsoleErrorConsumer);
-		gmConsole.setOutputConsumer(gmConsoleOutputConsumer);
 		diffReader = new DiffReaderXmlImpl();
 	}
 
 	public double compare(final String baseUseCaseName, final String baseScenarioName, final StepLink baseStepLink,
-			final String comparisonScreenshotName) {
-		if(!isGraphicsMagickAvailable()) {
-			return 0.0;
-		}
+						  final String comparisonScreenshotName) {
 
 		final String baseScreenshotName = THREE_DIGIT_NUM_FORMAT.format(baseStepLink.getStepIndex())
-				+ SCREENSHOT_FILE_EXTENSION;
-		final String diffScreenshotName = baseScreenshotName;
+			+ SCREENSHOT_FILE_EXTENSION;
 
 		final File baseScreenshot = docuReader.getScreenshotFile(baseBranchName,
-				baseBuildName, baseUseCaseName, baseScenarioName, baseScreenshotName);
+			baseBuildName, baseUseCaseName, baseScenarioName, baseScreenshotName);
 
 		final File comparisonScreenshot = docuReader.getScreenshotFile(
-				comparisonConfiguration.getComparisonBranchName(),
-				comparisonConfiguration.getComparisonBuildName(), baseUseCaseName, baseScenarioName,
-				comparisonScreenshotName);
+			comparisonConfiguration.getComparisonBranchName(),
+			comparisonConfiguration.getComparisonBuildName(), baseUseCaseName, baseScenarioName,
+			comparisonScreenshotName);
 
 		final File diffScreenshot = diffReader.getScreenshotFile(baseBranchName, baseBuildName,
-				comparisonConfiguration.getName(),
-				baseUseCaseName, baseScenarioName, diffScreenshotName);
+			comparisonConfiguration.getName(),
+			baseUseCaseName, baseScenarioName, baseScreenshotName);
 
-		if(!baseScreenshot.exists()) {
+		if (!baseScreenshot.exists()) {
 			LOGGER.warn("Base screenshot does not exist: " + baseScreenshot.getAbsolutePath());
 			return 0.0;
 		}
-		if(!comparisonScreenshot.exists()) {
+		if (!comparisonScreenshot.exists()) {
 			LOGGER.warn("Comparison screenshot does not exist: " + baseScreenshot.getAbsolutePath());
 			return 0.0;
 		}
 
 		return compareScreenshots(baseScreenshot, comparisonScreenshot, diffScreenshot);
 	}
 
-	boolean isGraphicsMagickAvailable() {
-		return GraphicsMagickConfiguration.isAvailable();
-	}
-
 	/**
-	 * A Diff Screenshoot will be created and stored in the directory of the diffScreenshot path.
+	 * A Diff Screenshot will be created and stored in the directory of the diffScreenshot path.
 	 * This directory, and all the parent directories, will be created if they do not exist.
 	 */
-	double compareScreenshots(final File baseScreenshot, final File comparisonScreenshot,
-			final File diffScreenshot) {
+	double compareScreenshots(final File baseScreenshot,
```

Comment:
Maybe not the best name, agree, but your name is not a better name for me - sorry - the proposed alternative would be less understandable for me to quickly understand what this method is doing - maybe something like `compareScreenshotAndStoreDiffImage` - creating parent directories for storing something is something we always do, whenever it is needed for storing something. So that is not realy an unexpected side effect here.